docs/installguide/release_notes.rst

```diff
@@ -310,15 +341,14 @@ Bug fixes
 General
 ^^^^^^^
 
-* KA Lite is officially supported on OSX 10.11 (El Capitan).
-* We've revamped the language packs into a new format, called content packs.
-  This results in significantly faster startup times across the board.
-.. WARNING::
-   You will have to redownload all your languages to fully support 0.16.
+ * KA Lite is officially supported on OSX 10.11 (El Capitan).
```

Comment:
Oh yes, maybe could you also add a PR for that? Would be great to have you take part in docs authoring!

Thanks for reading through it!